package.json

```diff
@@ -7,13 +7,11 @@
   "scripts": {
     "build": "npm run clean && webpack --progress --colors --bail",
     "clean": "rimraf ./dist && mkdirp dist && rimraf ./playground && mkdirp playground",
-    "deploy": "touch playground/.nojekyll && gh-pages -t -d playground -m \"[skip ci] Build for $(git log --pretty=format:%H -n1)\"",
     "i18n:push": "tx-push-src scratch-editor paint-editor ./translations/en.json",
     "i18n:src": "rimraf ./translations/messages && babel src > tmp.js && rimraf tmp.js && ./scripts/build-i18n-source.js ./translations/messages/ ./translations/",
     "lint": "eslint . --ext .js,.jsx",
-    "prepare": "husky install",
```

Comment:
Was this giving you trouble?

Reply:
Yep. I was getting an error on install. 

Looking `wagoid/commitlint-github-action@v5`  it looks like there is a husky dependency included, so I was thinking it wasn't necessary to install it as a pre-install, install